### ISSUE TITLE
RE: Windows 11 WSL2 Instance on Debian Distro...

### ISSUE THREAD

**Issue by silver2row** [ArcEye/machinekit-cnc#99]
Hello,

I did this one and then twice. Both times, each separate time, my computer started to receive these files that are not usable on Win 11 b/c of me installing MachineKit-CNC via APT on the WSL2 Debian Distro.

What is going on? 

Seriously, is there a major gaff going on and these random files from Debian just ` plow ` into Windows 11 WSL2 instances?

I really do not get it. I tried to erase the files from Win 11. Success! But...

` THE FILES KEPT COMING BACK! ` I cannot do anything to rid myself of the files so far. 

I am reaching out now b/c I would like the files off my Win 11 Instance, to have peace of mind back, and to know that WSL2 is not the ` enemy `.

Seth

P.S. It is my fault. I know. I trusted a bunch of people doing well with an Open Source ex-archived repo or two. Anyway, if there is a simplistic way to relieve myself of what I have done, please provide some type of guidance.

**Comment by silver2row** [comment]
Hello,

No issue now. I finally figured out how to remove the files but you guys...

1. Maybe a disclaimer would be nice. 
2. It killed the new Bullseye images from BBB.io.
3. I am done being upset.

Thank you.

Seth

**Comment by silver2row** [comment]
Hello to Anyone Reading These Closed Comments,

I am 100% positive that I know nothing about how hard you guys are working to make it happen with updated kernels and images people will produce. Nothing. But...

WIP or testing notices would have been nice.

Also...if you are using WSL2 on Win 11 via a Debian Distro (Bullseye 5.10.x), erasing via commands is a good thing to know. For some reason Win 11 via WSL2 Debian is creating a bunch of files on Win 11 start menu.

` Remove-Item -force TheFile.ini ` or whatever to remove the entire directory like so, ` Remove-Item -force C:\wherever_the_start_menu_folder_is_located/debian ` .

That will help to allow people using WSL2 instances a backbone of how not to panic.

Seth

P.S. Two times I panicked. I suffered both times. My fault. But...this may prove useful for people building and cross-compiling from WSL2 instances (Debian Bullseye 5.10.x).
